### DISCUSSION TITLE
AppCenter and Flathub

### DISCUSSION THREAD

**Discussion by volfpeter** [elementary/triage#386] [Ideas]
First of all, thanks for the great work!

The ideas (nothing new, but I haven't seen them in another discussion):

- It would be really great if the Flathub remote was added by default to AppCenter.
- It would be even greater if the Flathub approved badge was supported (in addition to the "verified by elementary" badge).
- And semantic colors for app info cards would improve AppCenter's UI quite a bit in my opinion (see image below).

![image](https://github.com/elementary/triage/assets/30116125/adafd202-75f0-4672-9797-4730d85abc0c)

**Comment by danirabbit** [comment]
I guess I’ll close this as completed because we do ship with Flathub out of the box these days :)
